embark-bibtex.el

```diff
@@ -76,7 +88,8 @@
 extracts the key from the candidate selected in embark and
 passes it to ACTION."
   `(defun ,name (cand)
-     (interactive "s ")
+     (interactive
+      (list (embark-bibtex--read "BibTeX entries: ")))
```

Comment:
Maybe you want to configure a different prompt here per embarkified command?

Reply:
`embark-bibtex--read` should not take a prompt argument at all, no?

Reply:
Oh, it should. For each action a separate prompt. This ensures that the commands are all usable in standalone moe.